BlogTemplate/wwwroot/css/site.css

```diff
@@ -76,3 +76,7 @@ footer {
 #Comment-Body{
     font: 16px/1 arial !important;
 }
+
+.deletedContent {
+    background-color: hsl(336, 100%, 76%);
+}
```

Comment:
nit: end files with a newline.  It makes for a better diff when you add more lines later.